backend/internal/data/sql/queries/shutter_explorer.sql

```diff
@@ -95,4 +95,15 @@ FROM decrypted_tx dt
 INNER JOIN transaction_submitted_event tse ON dt.transaction_submitted_event_id = tse.id
 WHERE dt.tx_status = 'included'
 ORDER BY dt.created_at
-LIMIT $1;
\ No newline at end of file
+LIMIT $1;
+
+-- name: QueryTotalRegisteredValidators :many
+SELECT COUNT(DISTINCT validator_index) AS total_registered_validators
```

Comment:
maybe the right place for the comment was here.

Reply:
didnt understand this comment.